app/models/victim_liaison_officer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class VictimLiaisonOfficer < ApplicationRecord
+  before_validation :omit_whitespace_on_email
```

Comment:
There is a generic/reusable pattern provided by a gem in the app which can do this without needing a custom method in the model. It was added with MO-308, so is only relatively new. I should probably have left a note in this ticket to point that out – sorry!

Here's an example how it's used in the `LocalDeliveryUnit` model:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/c763d5d961834961b9adbfc0c9148b54eac12c6d/app/models/local_delivery_unit.rb#L9

In this PR, it should be possible to replace the `before_validation` with `auto_strip_attributes`, and then we won't need the custom method `omit_whitespace_on_email` in this model. One less line of code for us to maintain 😄